### ISSUE TITLE
EntityPlayer class could not be found

### ISSUE THREAD

**Issue by Darky-BlackFang** [nickuc/OpeNLogin#53]
Hi, I install the plugin in my server but I see this thread/WARN in the console, and when this message appears in the console the server is a little laggy, How can i fix this?, Sorry for My bad English

**28.10 16:53:10 [Server] Server thread/WARN java.lang.ClassNotFoundException: net.minecraft.server.v1_17_R1.EntityPlayer
28.10 16:53:10 [Server] Server thread/WARN at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:142)
28.10 16:53:10 [Server] Server thread/WARN at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:99)
28.10 16:53:10 [Server] Server thread/WARN at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:519)
28.10 16:53:10 [Server] Server thread/WARN at java.base/java.lang.Class.forName0(Native Method)
28.10 16:53:10 [Server] Server thread/WARN at java.base/java.lang.Class.forName(Class.java:375)
28.10 16:53:10 [Server] Server thread/WARN at com.nickuc.openlogin.bukkit.reflection.ReflectionUtils.getNSNMS(ReflectionUtils.java:29)
28.10 16:53:10 [Server] Server thread/WARN at com.nickuc.openlogin.bukkit.reflection.packets.Packet.<clinit>(Packet.java:27)
28.10 16:53:10 [Server] Server thread/WARN at com.nickuc.openlogin.bukkit.listeners.PlayerJoinListeners.onPlayerJoin(PlayerJoinListeners.java:64)
28.10 16:53:10 [Server] Server thread/WARN at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
28.10 16:53:10 [Server] Server thread/WARN at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
28.10 16:53:10 [Server] Server thread/WARN at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
28.10 16:53:10 [Server] Server thread/WARN at java.base/java.lang.reflect.Method.invoke(Method.java:567)
28.10 16:53:10 [Server] Server thread/WARN at org.bukkit.plugin.java.JavaPluginLoader$1.execute(JavaPluginLoader.java:306)
28.10 16:53:10 [Server] Server thread/WARN at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70)
28.10 16:53:10 [Server] Server thread/WARN at org.bukkit.plugin.SimplePluginManager.fireEvent(SimplePluginManager.java:589)
28.10 16:53:10 [Server] Server thread/WARN at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:576)
28.10 16:53:10 [Server] Server thread/WARN at net.minecraft.server.players.PlayerList.a(PlayerList.java:276)
28.10 16:53:10 [Server] Server thread/WARN at net.minecraft.server.network.LoginListener.a(LoginListener.java:192)
28.10 16:53:10 [Server] Server thread/WARN at net.minecraft.server.network.LoginListener.c(LoginListener.java:179)
28.10 16:53:10 [Server] Server thread/WARN at net.minecraft.server.network.LoginListener.tick(LoginListener.java:74)
28.10 16:53:10 [Server] Server thread/WARN at net.minecraft.network.NetworkManager.a(NetworkManager.java:246)
28.10 16:53:10 [Server] Server thread/WARN at net.minecraft.server.network.ServerConnection.c(ServerConnection.java:172)
28.10 16:53:10 [Server] Server thread/WARN at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1336)
28.10 16:53:10 [Server] Server thread/WARN at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:438)
28.10 16:53:10 [Server] Server thread/WARN at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1217)
28.10 16:53:10 [Server] Server thread/WARN at net.minecraft.server.MinecraftServer.x(MinecraftServer.java:1050)
28.10 16:53:10 [Server] Server thread/WARN at net.minecraft.server.MinecraftServer.lambda$0(MinecraftServer.java:305)
28.10 16:53:10 [Server] Server thread/WARN at java.base/java.lang.Thread.run(Thread.java:831)**

**Comment by nickuc** [comment]
Hi, @Darky-BlackFang!

Try to use the latest version of Paper:
https://papermc.io/api/v2/projects/waterfall/versions/1.17/builds/451/downloads/waterfall-1.17-451.jar

**Comment by Caticer** [comment]
I have same ward message
(I have latest version paper 1.18 and plugin)
```[15:53:24 WARN]: java.lang.ClassNotFoundException: net.minecraft.server.v1_18_R1.EntityPlayer
[15:53:24 WARN]:        at org.bukkit.plugin.java.PluginClassLoader.loadClass0(PluginClassLoader.java:151)
[15:53:24 WARN]:        at org.bukkit.plugin.java.PluginClassLoader.loadClass(PluginClassLoader.java:103)
[15:53:24 WARN]:        at java.base/java.lang.ClassLoader.loadClass(ClassLoader.java:520)
[15:53:24 WARN]:        at java.base/java.lang.Class.forName0(Native Method)
[15:53:24 WARN]:        at java.base/java.lang.Class.forName(Class.java:375)
[15:53:24 WARN]:        at OpenLogin.jar//com.nickuc.openlogin.bukkit.reflection.ReflectionUtils.getNSNMS(ReflectionUtils.java:29)
[15:53:24 WARN]:        at OpenLogin.jar//com.nickuc.openlogin.bukkit.reflection.packets.Packet.<clinit>(Packet.java:28)
[15:53:24 WARN]:        at OpenLogin.jar//com.nickuc.openlogin.bukkit.listeners.PlayerJoinListeners.onPlayerJoin(PlayerJoinListeners.java:64)
[15:53:24 WARN]:        at com.destroystokyo.paper.event.executor.asm.generated.GeneratedEventExecutor75.execute(Unknown Source)
[15:53:24 WARN]:        at org.bukkit.plugin.EventExecutor.lambda$create$1(EventExecutor.java:69)
[15:53:24 WARN]:        at co.aikar.timings.TimedEventExecutor.execute(TimedEventExecutor.java:80)
[15:53:24 WARN]:        at org.bukkit.plugin.RegisteredListener.callEvent(RegisteredListener.java:70)
[15:53:24 WARN]:        at org.bukkit.plugin.SimplePluginManager.callEvent(SimplePluginManager.java:628)
[15:53:24 WARN]:        at net.minecraft.server.players.PlayerList.postChunkLoadJoin(PlayerList.java:359)
[15:53:24 WARN]:        at net.minecraft.server.players.PlayerList.lambda$placeNewPlayer$1(PlayerList.java:301)
[15:53:24 WARN]:        at net.minecraft.server.network.PlayerConnection.b(PlayerConnection.java:306)
[15:53:24 WARN]:        at net.minecraft.network.NetworkManager.a(NetworkManager.java:526)
[15:53:24 WARN]:        at net.minecraft.server.network.ServerConnection.c(ServerConnection.java:201)
[15:53:24 WARN]:        at net.minecraft.server.MinecraftServer.b(MinecraftServer.java:1636)
[15:53:24 WARN]:        at net.minecraft.server.dedicated.DedicatedServer.b(DedicatedServer.java:480)
[15:53:24 WARN]:        at net.minecraft.server.MinecraftServer.a(MinecraftServer.java:1470)
[15:53:24 WARN]:        at net.minecraft.server.MinecraftServer.w(MinecraftServer.java:1264)
[15:53:24 WARN]:        at net.minecraft.server.MinecraftServer.lambda$spin$0(MinecraftServer.java:317)
[15:53:24 WARN]:        at java.base/java.lang.Thread.run(Thread.java:833)```

**Comment by nickuc** [comment]
Hi! 

Could you upload your server jar? I will try to repeat the error.

**Comment by Caticer** [comment]
[Here](https://easyupload.io/oizfuk)

**Comment by DaiveeCZ** [comment]
Same problem here, using Paper. After update to 1.18 same problem. It shows only on first  join after restart server (player - each warn). Login is a bit laggy, but in 1.17 works normal - maybe paper 1.18 (experimental) issue .

**Comment by nickuc** [comment]
Fixed in 1.5
